react/SelectBox/SelectBox.jsx

```diff
@@ -45,6 +45,12 @@ const customStyles = props => ({
   menu: base => ({
     ...base,
     zIndex: 10
+  }),
+  menuList: base => ({
+    ...base,
+    maxHeight: props.numLines
+      ? `${props.numLines * 2.125 + 0.25}rem`
```

Comment:
What is `2.125 + 0.25` ? Height of a line + padding ?

Reply:
It misses a comment, my bad (but you're right).